app/Http/Controllers/ConfigController.php

```diff
@@ -71,50 +80,43 @@ public function handleConfiguration(Request $request)
                     'stores/' . $this->getStoreHash() . '/v3/content/scripts/' . $uuid
                 );
                 $scriptRow->delete();
-            } else {
-                // checking that active button is checked then add script
-                if (isset($activeStatus)) {
-                    $jsSnippet = view('jsSnippet', [
-                        'shopkey' => $shopkey
-                    ]);
-
-                    $requestBody = [
-                        'name' => 'Findologic',
-                        'description' => 'Search & Navigation Platform',
-                        'html' => $jsSnippet->render(),
-                        'auto_uninstall' => true,
-                        'load_method' => 'default',
-                        'location' => 'head',
-                        'visibility' => 'storefront',
-                        'kind' => 'script_tag',
-                        'consent_category' => 'essential',
-                    ];
-
-                    $response = $this->makeBigCommerceAPIRequest(
-                        'POST',
-                        'stores/' . $this->getStoreHash() . '/v3/content/scripts',
-                        json_encode($requestBody)
-                    );
-                    // converting data
-                    $body = $response->getBody()->getContents();
-                    $collection = collect(json_decode($body, true));
-
-                    $data = $collection['data'];
-                    // saving in database
-                    $script = new Script;
-                    $script->name = $data['name'];
-                    $script->uuid = $data['uuid'];
-                    $script->store_hash = $storeHash;
-                    $script->save();
-                }
             }
 
-        }
+            // Add new snippet with fresh config
```

Comment:
Instead of adding a comment, describing what a code block does, why not put this into a separate method?